.gitignore

```diff
@@ -82,3 +82,5 @@ logs/*.out
 
 .sbt/
 .ivy2/
+*.plan
+*.js
```

Comment:
Why did you add `.js` if we do have some `js` files?

Reply:
The reason is that the command `bin/sync-profiles` will generate `.js` for `ci.yml`, which is never needed.